### DISCUSSION TITLE
Reloading page in Angular 14 logs me out every time

### DISCUSSION THREAD

**Discussion by shamusschubert** [abacritt/angularx-social-login#569] [Q&A]
The flow is very odd to me.  When I first load the app, I get the Google login popup in the upper left corner automatically, but if I close that out there is no way to get it back.  

Next, if I do login that way, it seems to persist from page to page when navigating, but when I 'reload' the same page I am logged out, and that Google box pops back up again.  

I can add the Google login button on the home screen (app.component.html page) but ideally would like to move all this to the navbar instead like most apps.

**Comment by terrarum** [comment]
Same experience here. I just updated to 1.2.3 and now reloading any page in my app or opening a link in a new tab restarts the login process.

**Comment by taneljoeaar** [comment]
Not sure if this is helping, but try adding `data-auto_select="true"` to your onload prompt. This makes the prompt automatically login user again on reload, like on the demo.
https://developers.google.com/identity/gsi/web/guides/automatic-sign-in-sign-out

**Reply by taneljoeaar** [reply]
But yeah, I'd too wish it worked the old way.

**Reply by IanaPushcash** [reply]
I don't understand where should I add this data-auto_select="true". Could you specify a piece of code?
